nvflare/private/fed/simulator/simulator_server.py

```diff
@@ -36,6 +36,10 @@ def sync_clients_from_main_process(self):
     def update_job_run_status(self):
         pass
 
+    def fire_event(self, event_type: str, fl_ctx: FLContext):
+        if self.run_manager:
+            self.run_manager.fire_event(event_type, fl_ctx)
```

Comment:
would this affect current simulator server behavior

Reply:
This does not affect the current simulator.